src/components/FormPairRow.tsx

```diff
@@ -107,16 +94,10 @@ const MultiTags: React.FC<IPairRow> = ({ item }) => {
         </div>
       ))}
 
-      {isLST && (
```

Comment:
Why we remove lst tag?

Reply:
we should standardise a "verified" tag instead of having so many tags that user might be confused.